scripts/pylib/twister/twisterlib/testsuite.py

```diff
@@ -265,6 +296,8 @@ def scan_testsuite_path(testsuite_path):
     ztest_suite_names = []
 
     src_dir_path = _find_src_dir_path(testsuite_path)
+
+    testsuites = {}
```

Comment:
Do you need this `testsuites` variable for something? Beyond collecting only some data it seems to be unused in any further logic.

Reply:
yes, this will be used down the road for better test management and labeling, right now I am using it for debugging, but it s too verbose, even for `logger.debug()`